1st two matches from 2021 "base-space tournament"

jordancurve opened the pull request:
Add Burning Skull's openings from his first two matches in the 2021 "Dominion base-space tournament".  Sources:
- Burning Skull, [Dominion base-space tournament vs amoffett11](https://www.youtube.com/watch?v=DorVPvPQQxY), *YouTube* (2021-03-28)
- Burning Skull, [Dominion base-space tournament vs JaCrispyKing](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-04)

jordancurve committed:
1st two matches from 2021 "base-space tournament"

Adds Burning Skull's openings from his first two matches in the 2021 "Dominion base-space tournament".  Sources:
- Burning Skull, [Dominion base-space tournament vs amoffett11](https://www.youtube.com/watch?v=DorVPvPQQxY), *YouTube* (2021-03-28)
- Burning Skull, [Dominion base-space tournament vs JaCrispyKing](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-04)

zuirod reviewed (changes requested):
Hey, great idea! Thanks for your contribution. I have a couple of questions:
1. Burning Skull just finished his third and final match. Could you add that?
2. I see two openings for game id 70580237, so one of those should have hypothetical set to true? (hypothetical meaning that he said he would play it, but didn't actually play it in the game)
3. There are 4 games per match, but only 5 new game ids. Which games did you not include?

jordancurve committed:
1st 3 matches from 2021 "base-space tournament"

Add Burning Skull's openings from his first 3 matches in the 2021 "Dominion base-space tournament". Sources:

Burning Skull, [Dominion base-space tournament vs amoffett11](https://www.youtube.com/watch?v=DorVPvPQQxY), *YouTube* (2021-03-28)
Burning Skull, [Dominion base-space tournament vs JaCrispyKing](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-04)
Burning Skull, [Dominion base-space tournament vs JNails](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-14)

Below are links with timestamps to the start of each game and to a frame that shows the opening. This should make it easier to check this PR for mistakes.

## vs amoffett11

70580237: [start](https://youtu.be/DorVPvPQQxY?t=22) / [opening](https://youtu.be/DorVPvPQQxY?t=108)
70580796: [start](https://youtu.be/DorVPvPQQxY?t=662) / [opening](https://youtu.be/DorVPvPQQxY?t=748)
70581749: [start](https://youtu.be/DorVPvPQQxY?t=1700) / [opening](https://youtu.be/DorVPvPQQxY?t=1813)
70582512: [start](https://youtu.be/DorVPvPQQxY?t=2537) / [opening](https://youtu.be/DorVPvPQQxY?t=2597)

## vs JaCrispyKing

71112172: [start](https://youtu.be/rgmGwpqKpYs?t=29) / [opening](https://youtu.be/rgmGwpqKpYs?t=87)
71113715: [start](https://youtu.be/rgmGwpqKpYs?t=1905) / [opening](https://youtu.be/rgmGwpqKpYs?t=1974)
71114568: [start](https://youtu.be/rgmGwpqKpYs?t=2980) / [opening](https://youtu.be/rgmGwpqKpYs?t=3083)
71115823: [start](https://youtu.be/rgmGwpqKpYs?t=4360) / [opening](https://youtu.be/rgmGwpqKpYs?t=4537)

## vs JNails

71933307: [start](https://youtu.be/o_WBTSKAXiA?t=0) / [opening](https://youtu.be/o_WBTSKAXiA?t=131)
71934132: [start](https://youtu.be/o_WBTSKAXiA?t=610) / [opening](https://youtu.be/o_WBTSKAXiA?t=786)
71934929: [start](https://youtu.be/o_WBTSKAXiA?t=1179) / [opening](https://youtu.be/o_WBTSKAXiA?t=1242)
71935274: [start](https://youtu.be/o_WBTSKAXiA?t=1440) / [opening](https://youtu.be/o_WBTSKAXiA?t=1511)

jordancurve commented:
Oops, thanks for catching that. I've updated the PR to add the final match, fix the duplicate game ID, and add the missing games. I also added some extra links to the description make it easier to check my work.

zuirod commented on data/openings.csv:
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
 6227420,1,2,5-2,Sentry-,Merchant-Gardens-Militia-Moneylender-Remodel-Throne Room-Bandit-Festival-Laboratory-Sentry
 70580237,0,1,3-4,Silver-Militia,Cellar-Moat-Vassal-Village-Militia-Poacher-Festival-Library-Sentry-Witch
-70580237,0,1,2-5,Chapel-Laboratory,Chapel-Moat-Workshop-Bureaucrat-Moneylender-Poacher-Throne Room-Bandit-Laboratory-Witch
+70580796,0,1,2-5,Chapel-Laboratory,Chapel-Moat-Workshop-Bureaucrat-Moneylender-Poacher-Throne Room-Bandit-Laboratory-Witch
```
player should be 2 here

zuirod commented on data/openings.csv:
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
+70581749,0,1,4-3,Moneylender-Silver,Moat-Vassal-Militia-Moneylender-Poacher-Remodel-Council Room-Festival-Market-Mine
 70582512,0,2,4-3,Chapel-Vassal,Chapel-Harbinger-Vassal-Throne Room-Laboratory-Market-Mine-Sentry-Witch-Artisan
 71112172,0,1,3-4,Vassal-Poacher,Vassal-Village-Bureaucrat-Poacher-Smithy-Throne Room-Council Room-Festival-Library-Sentry
 71113715,0,2,4-3,Poacher-Chapel,Cellar-Chapel-Merchant-Vassal-Village-Poacher-Bandit-Laboratory-Market
```
Kingdom missing Artisan

zuirod commented on data/openings.csv:
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
 70582512,0,2,4-3,Chapel-Vassal,Chapel-Harbinger-Vassal-Throne Room-Laboratory-Market-Mine-Sentry-Witch-Artisan
 71112172,0,1,3-4,Vassal-Poacher,Vassal-Village-Bureaucrat-Poacher-Smithy-Throne Room-Council Room-Festival-Library-Sentry
 71113715,0,2,4-3,Poacher-Chapel,Cellar-Chapel-Merchant-Vassal-Village-Poacher-Bandit-Laboratory-Market
+71114568,0,1,4-3,Militia-Chapel,Cellar-Chapel-Moat-Village-Gardens-Militia-Moneylender-Smithy-Laboratory-Sentry
```
Sentry should be Library

zuirod commented on data/openings.csv:
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
+71114568,0,1,4-3,Militia-Chapel,Cellar-Chapel-Moat-Village-Gardens-Militia-Moneylender-Smithy-Laboratory-Sentry
 71115823,0,2,3-4,Silver-Militia,Moat-Merchant-Vassal-Workshop-Gardens-Militia-Library-Market-Sentry-Artisan
+71933307,0,1,5-2,Bandit-Chapel,Cellar-Chapel-Merchant-Vassal-Moneylender-Smithy-Bandit-Council Room-Laboratory-Library
+71934132,0,2,5-2,Workshop-Moat,Chapel-Moat-Merchant-Village-Workshop-Bureaucrat-Gardens-Poacher-Smithy-Library-Mine
```
11 cards in the kingdom. Village should be removed.

zuirod commented on data/openings.csv:
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
+71933307,0,1,5-2,Bandit-Chapel,Cellar-Chapel-Merchant-Vassal-Moneylender-Smithy-Bandit-Council Room-Laboratory-Library
+71934132,0,2,5-2,Workshop-Moat,Chapel-Moat-Merchant-Village-Workshop-Bureaucrat-Gardens-Poacher-Smithy-Library-Mine
+71934929,0,1,3-4,Chapel-Militia,Chapel-Harbinger-Militia-Moneylender-Poacher-Remodel-Council Room-Laboratory-Library-Market
+71935274,0,1,3-4,Chapel-Poacher,Chapel-Harbinger-Merchant-Gardens-Poacher-Smithy-Bandit-Laboratory-Market-Witch
```
player should be 2

zuirod reviewed (changes requested):
Thanks for adding the final match!

jordancurve committed:
1st 3 matches from 2021 "base-space tournament"

Add Burning Skull's openings from his first 3 matches in the 2021 "Dominion base-space tournament". Sources:

Burning Skull, [Dominion base-space tournament vs amoffett11](https://www.youtube.com/watch?v=DorVPvPQQxY), *YouTube* (2021-03-28)
Burning Skull, [Dominion base-space tournament vs JaCrispyKing](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-04)
Burning Skull, [Dominion base-space tournament vs JNails](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-14)

Below are links with timestamps to the start of each game and to a frame that shows the opening. This should make it easier to check this PR for mistakes.

## vs amoffett11

70580237: [start](https://youtu.be/DorVPvPQQxY?t=22) / [opening](https://youtu.be/DorVPvPQQxY?t=108)
70580796: [start](https://youtu.be/DorVPvPQQxY?t=662) / [opening](https://youtu.be/DorVPvPQQxY?t=748)
70581749: [start](https://youtu.be/DorVPvPQQxY?t=1700) / [opening](https://youtu.be/DorVPvPQQxY?t=1813)
70582512: [start](https://youtu.be/DorVPvPQQxY?t=2537) / [opening](https://youtu.be/DorVPvPQQxY?t=2597)

## vs JaCrispyKing

71112172: [start](https://youtu.be/rgmGwpqKpYs?t=29) / [opening](https://youtu.be/rgmGwpqKpYs?t=87)
71113715: [start](https://youtu.be/rgmGwpqKpYs?t=1905) / [opening](https://youtu.be/rgmGwpqKpYs?t=1974)
71114568: [start](https://youtu.be/rgmGwpqKpYs?t=2980) / [opening](https://youtu.be/rgmGwpqKpYs?t=3083)
71115823: [start](https://youtu.be/rgmGwpqKpYs?t=4360) / [opening](https://youtu.be/rgmGwpqKpYs?t=4537)

## vs JNails

71933307: [start](https://youtu.be/o_WBTSKAXiA?t=0) / [opening](https://youtu.be/o_WBTSKAXiA?t=131)
71934132: [start](https://youtu.be/o_WBTSKAXiA?t=610) / [opening](https://youtu.be/o_WBTSKAXiA?t=786)
71934929: [start](https://youtu.be/o_WBTSKAXiA?t=1179) / [opening](https://youtu.be/o_WBTSKAXiA?t=1242)
71935274: [start](https://youtu.be/o_WBTSKAXiA?t=1440) / [opening](https://youtu.be/o_WBTSKAXiA?t=1511)

jordancurve commented:
Done! Thanks for the corrections.

zuirod reviewed (approved):
Great!
